### PR TITLE
Add rabbitmqctl command for node health check

### PR DESCRIPTION
**Pull request by dcorbacho**
Part of #398

### CODE REVIEW

File: src/rabbit_control_main.erl [hunk 1]
```diff
@@ -547,6 +549,16 @@ action(eval, Node, [Expr], _Opts, _Inform) ->
...
     io:format("~s", [rabbit_ctl_usage:usage()]);
 
+action(node_health_check, Node, _Args, _Opts, Inform) ->
+    Inform("Health check of node ~p", [Node]),
+    try
+        rabbit_health_check:node(Node),
+        io:format("Node ~p is up and running~n", [Node])
+    catch
+        {node_is_ko, ErrorMsg} ->
+            io:format("~s~nProblems encountered in node ~p~n", [ErrorMsg, Node])
```
Comment (michaelklishin) [inline src/rabbit_control_main.erl]: @binarin what UNIX return code should we use when a check fails?

Comment (binarin) [inline src/rabbit_control_main.erl]: Currently there is only one possible reason - `node_is_ko`. EX_SOFTWARE (70) suits it the best.

And is it possible to add separate reason with exit code EX_NOHOST (68) for cases when a node is unavailable?

Comment (michaelklishin) [inline src/rabbit_control_main.erl]: @binarin good idea

**Comment by dcorbacho** [comment]
@michaelklishin Just added the unix error codes to rabbitmqctl

**Pull request merged**